Sources/SwiftCurrent_SwiftUI/Extensions/SwiftUILaunchStyleAdditions.swift

```diff
@@ -0,0 +1,59 @@
+//  swiftlint:disable:this file_name
+//  SwiftUILaunchStyleAdditions.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Tyler Thompson on 8/22/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import SwiftCurrent
+
+extension LaunchStyle {
```

Comment:
We may need to have a DSL like: `.navigationLink(.splitScreen)` or thereabouts to handle the additional styling you can put on a `NavigationLink`.

Reply:
Just looked it up and the options are: `.automatic`, `.columns`, and `.stack`